docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1583,8 +1583,20 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
   +<Tabs scrollButtons={false} />
   ```
 
+### Tab
+
 - Tab `minWidth` changed from `72px` => `90px` (without media-query) according to [material-design spec](https://material.io/components/tabs#specs)
 - Tab `maxWidth` changed from `264px` => `360px` according to [material-design spec](https://material.io/components/tabs#specs)
+- `span` element that wraps children has been removed. `wrapper` classKey is also removed. More details about [this change](https://github.com/mui-org/material-ui/pull/26666).
```

Comment:
The passive voice doesn't feel right. Most of the items in this guide are in the active voice.

```suggestion
- Change the Tab `minWidth` from `72px` => `90px` (without media-query) according to [material-design spec](https://material.io/components/tabs#specs)
- Change the Tab `maxWidth` from `264px` => `360px` according to [material-design spec](https://material.io/components/tabs#specs)
- Remove the `span` element that wraps children. Remove the `wrapper` classKey too. More details about [this change](https://github.com/mui-org/material-ui/pull/26666).
```

cc @mbrookes for confirmation.

Reply:
I think it's confusing linking different PR here. Let's update the current PR description with the details of why the changes was introduced and link this PR in the migration guide.

Reply:
> The passive voice doesn't feel right.

I'm not so sure. "Change the Tab ` minWidth`" sounds like an instruction to the reader, whereas "Tab `minWidth` (was/has been) changed" describes what has changed:

- What changed
- What is/are the implication(s)
- What you need to do

(Situation, Complication, Resolution...)

Reply:
Intuitively I would've also preferred passive voice in changelogs. 

Though it's not uncommon for changelogs to have active voice:

- https://github.com/facebook/react/blob/master/CHANGELOG.md
- https://keepachangelog.com/en/1.0.0/

changelogs with passive voice: 
- https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_0_1-release-notes

I could go on forever and find examples for active and passive voice. Though I can't find any resources on active vs passive voice for changelogs specifically (this is why "benchmarking" is generally useless to form an actionable advise). 

I'd say it's more important to use a consistent voice throughout the document since I'm fine with either voice. Though definitely an interesting topic.

Reply:
Thanks for the two examples. Would it work if we use this framing? 

1. In the changelog, the PR's description are in the active voice because they describe the action of the change. It's more git focused. 
2. In the changelog's summary, the passive voice is used, because we focus on the developers not familiar with the codebase reading it.
3. In the migration guide, the passive voice is used for the same reason as 3.

Reply:
similar thought as Sebastian, I am okay with the above proposal as long as we keep them consistent.